elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/models/Metric.java

```diff
@@ -43,6 +43,10 @@ public Metric(Class<?> tableClass, String fieldName, AggregationDictionary dicti
         } else {
             try {
                 this.metricFunction = metric.function().newInstance();
+                metricFunction.setName(getId() + "[" + metricFunction.getName() + "]");
```

Comment:
You can remove the else clause above (but keep the code block) and remove one level of nesting.